styles/styles.scss

```diff
@@ -1,12 +1,20 @@
-html {
-    width: 100%;
-    height: 100%;
+:root {
+  --navbar-bg-color: #ffffff;
+  --navbar-link-color: rgba(51, 51, 51, 0.7);
+  --navbar-link-hover-color: #333333;
+  --navbar-link-active-color: #333333;
+  --spacer-small: 0.5rem;
+  --spacer-default: 1rem;
+  --spacer-navbar-y: 3.5rem;
+  --spacer-navbar-x: 11.9375rem;
```

Comment:
Those two values should not be here. Spacer is the name we use for more reusable spacing. Specific styles like this should be given in component. But, it's something wrong with this value, we'll look on it on the call.